run/rails-cat_album/Gemfile.lock

```diff
@@ -95,15 +127,20 @@ GEM
     mail (2.7.1)
       mini_mime (>= 0.1.1)
     marcel (1.0.1)
+    memoist (0.16.2)
     method_source (1.0.0)
+    mime-types (3.3.1)
+      mime-types-data (~> 3.2015)
+    mime-types-data (3.2021.0225)
     mini_mime (1.0.3)
-    mini_portile2 (2.5.3)
     minitest (5.14.4)
     msgpack (1.4.2)
+    multi_json (1.15.0)
+    multipart-post (2.1.1)
     nio4r (2.5.7)
-    nokogiri (1.11.6)
-      mini_portile2 (~> 2.5.0)
+    nokogiri (1.11.6-x86_64-darwin)
```

Comment:
We need to change this not to be darwin-specific. Can you `bundle update --add-platform ruby` and then `bundle update --remove-platform x86_64-darwin`?